### PR TITLE
Configure recipients for yarn.lock changes in PRs

### PR DESCRIPTION
**Pull request by pedropombeiro**
This PR enables the Probot that will ping people whenever changes to yarn.lock are detected.

status: ready

### CODE REVIEW

**Review by jakubgs — APPROVED** [review]
Awesome change.

File: .github/package-lock-snitch.config.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "recipients": [
+        "jakubgs",
+        "mandrigin"
```
Comment (mandrigin) [inline .github/package-lock-snitch.config.json]: please add @corpetty there too

**Review by mandrigin — CHANGES_REQUESTED** [review]
one small thing

File: .github/package-lock-snitch.config.json (hunk 1 above)
Comment (pedropombeiro) [reply .github/package-lock-snitch.config.json]: Done

**Review by mandrigin — APPROVED** [review]

**Comment by corpetty** [comment]
thank you for adding me.

**Pull request merged**